### ISSUE TITLE
Button transitions for calculate features

### ISSUE THREAD

**Issue by jkr32** [CS3398-Tiger-Terrors/CS3398-Tiger-Terrors-S2017#100]
This will be the actual transition from clicking the calculate button that will show you the results.
Not a slide transition with your finger.

**Comment by jkr32** [comment]
This was my entire sprint, went from trying to make buttons work, to trying to make our one and only button work. Our one button has the entire app on its back. Donald took all the formulas that the team researched and formed a bunch of key strings and tried to implement a bundle Intent in andriod studio to pass them all as the one button was clicked that would toss you to the Results page. We did realize before this sprint that the core of our app (this part) was not functional at all, we just thought it was missing some stuff. I spent way over 1 spring, maybe about 40 hours trying to figure out why this button was not working. I tried android refactoring the code. I tried implementing a c++ doc last minute and passing the objects through JNI with a controller. I tried breaking the activity into fragments, which are basically sub activities inside of an activity, which an activity is basically what you see on each load screen of the app. So this would help by making each calculation a fragment of the initial Activity. At once point I commented out all of the formula classes part of the code and force implemented the transition in the button where that was all it did and it worked, kind of, after you clicked it and it passed to second activity it auto crashed app even though it was literally passing nothing. This process should not of been hard. you make a public button, you set view of button to where to find it (R.id.nameofbutton) then you set up the Intent, and startActivity. We took this and tried to parse it with classes making all the calculations a bundle inside the public void for the buttonclick and then pass the bundle as the Intent for activity start. This made the button simply un-clickable.

**Comment by jkr32** [comment]
Something happen, I guess bad communication but our development branch code got corrupted. Maybe someone pushed bad code, not really sure. But after hours of texting and having to call each other Donald and I finally found a working directory inside of his personal local branch of code that still kind of worked. So after everything we did not get the project to work at the end which hurt most people in the sprint because a lot of tasks were depending on a working app. I only just now gave up because I have slept yet and can barely keep my eyes open. I even made a last resort attempt to start completely over from scratch and redo all the formulas but there was just no way I had enough time to pull it off.
